std/ascii.zig

```diff
@@ -195,7 +283,11 @@ pub fn isASCII(c: u8) bool {
 }
 
 pub fn isBlank(c: u8) bool {
-    return (c == ' ') or (c == '\x09');
+    return inTable(c, tIndex.Blank);
+}
+
+pub fn isZig(c: u8) bool {
```

Comment:
What is this? Can you document this better?

Reply:
It is documented elsewhere in the file, but I pushed some doc friendly documentation. I was using it in the self-hosted compiler to validate zig source code encoding. (plus other stuff to validate the utf-8/unicode)